docs/06-contributors/999-rfcs/000-stories/story-15.md

```diff
@@ -56,76 +57,78 @@ A good example on how this may look like is Redis embedded prompt inside their d
 bring cloud;
 bring redis;
 
-extern "./tasklist_helper.js" {
-  inflight get_data: (url: str) => Json;
-  inflight create_regex: (s: str) => IMyRegExp;
-  inflight uuid() => str; 
-};
+interface IMyRegExp {
+  inflight test(s: str): bool
+}
 
 enum Status {
   Uncompleted,
   Completed
 }
 
-interface IMyRegExp {
-  inflight test(s: str): bool
-}
-
 interface ITaskList {
   inflight get(id: str): Json;
   inflight add(title: str): str;
   inflight remove(id: str): void; 
-  inflight find(r: str): Array<str>;
+  inflight find(r: IMyRegExp): Array<str>;
   inflight set_status(id: str, status: Status): str;
   inflight set_estimation(id: str, estimation: duration): str;
 }
 
 resource TaskList implementes ITaskList {
   _redis: redis.Redis;
+  // we are missing inflight init, so I need to create a lazy get_redis_client method  
+  // that creates the _redis_client when it is first called
+  inflight var _redis_client: redis.IRedisClient?; 
```

Comment:
I think we don't support inflight fields yet: [Resource Declaration: inflight fields](https://github.com/winglang/wing/issues/1596)

Reply:
the following code works on 0.5.117

```ts (wing)
bring cloud;

resource A{
  inflight var name: str;
  init() {
  }
}

let a = new A();

new cloud.Function(inflight () => {
  a.name = "hello";
  assert("hello" == a.name);
}) as "test";
```

Reply:
@staycoolcall911 , what do you say?

Reply:
I'll ask @yoav-steinberg later today on our team meeting

Reply:
The consensus on our team was that this works. Keep going 💪